pkg/testutils/testutils.go

```diff
@@ -382,6 +382,26 @@ func RecoverHNC() {
 	cleanupNamespaces(a, b)
 }
 
+func AddDisableEnforcedTypesArg() {
+	// If hncRecoverPath is not set, we should not get here, but if we do, return early and - propably - fail the tests
+	// as the changes done in this prep function require the original manifest file to recover a "normal" state.
+	if hncRecoverPath == "" {
+		return
+	}
+
```

Comment:
As long as the test calls `CheckHNCPath()` before calling this, we shouldn't need these lines. There are many cases where this _isn't_ set and we don't want the tests to fail.